### PR TITLE
[add] system_type column to classification sets

### PR DESCRIPTION
**Pull request by andres-martin**
Create a query migration to add the system_type column to classification sets

https://fulcrumapp.atlassian.net/browse/FLCRM-8125

### CODE REVIEW

File: sql/template.sql [hunk 1]
```diff
@@ -88,6 +88,7 @@ CREATE TABLE IF NOT EXISTS organization.classification_sets (
   created_by_resource_id text,
   updated_by_id bigint,
   updated_by_resource_id text,
+  system_type text NOT NULL,
```
Comment (toddrun) [inline sql/template.sql]: Are you sure this can't be null???? Seems wrong to me.

File: src/schemas/postgres/classification-sets.js [hunk 2]
```diff
@@ -17,6 +17,7 @@ export default class ClassificationSets extends TableDefinition {
     this.string('created_by_resource_id', {});
     this.integer('updated_by_id', {});
     this.string('updated_by_resource_id', {});
+    this.string('system_type', {"allowNull":false});
```
Comment (toddrun) [inline src/schemas/postgres/classification-sets.js]: This too seems wrong

File: src/schemas/v2.js [hunk 3]
```diff
@@ -762,6 +762,10 @@ Schema.systemClassificationSetsTable = [
     name: 'updated_at',
     type: 'timestamp',
     allowNull: false
+  }, {
+    name: 'system_type',
+    type: 'string',
+    allowNull: false
```
Comment (toddrun) [inline src/schemas/v2.js]: Ditto to other comments.

**Review by toddrun — COMMENTED** [review]
Pretty sure system_type needs to be nullable. Since I'm not 100% sure, I'm neither approving or requesting changes . . . just asking for clarification and review from others who can confirm/deny whether the field should be nullable.

File: sql/template.sql (hunk 1 above)
Comment (andres-martin) [reply sql/template.sql]: looking at other system_type columns they seem to be nullable as well

**Review by pedrodotavila — APPROVED** [review]
Looks good to me 😄

**Review by canriquezds — APPROVED** [review]

**Pull request merged**